src/main/java/se/sundsvall/contract/service/ContractService.java

```diff
@@ -0,0 +1,37 @@
+package se.sundsvall.contract.service;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import se.sundsvall.contract.api.model.ContractHolder;
+import se.sundsvall.contract.api.model.ContractRequest;
+import se.sundsvall.contract.api.model.LandLeaseContract;
+
+@Service
+public class ContractService {
+
```

Comment:
Rename methods to what they do instead of what the client (of this class) do?
The service class can be used in other scenarios than just backing a REST-controller.

e.g.
postContract -> createContract
patchContract -> updateContract
and so on...